cephfs/cephfs.go

```diff
@@ -36,40 +36,36 @@ func CreateMount() (*MountInfo, error) {
 	ret := C.ceph_create(&mount.mount, nil)
 	if ret == 0 {
```

Comment:
```golang
if ret != 0 {
                 log.Errorf("CreateMount: Failed to create mount")
		return nil, CephError(ret)
}
return mount, nil
```

Reply:
What is the purpose of switching the the contents of the if statement?

Reply:
https://github.com/kubernetes/kubernetes/blob/f883fd2ce6c9c41d2db7c752a6f193b5fa928205/staging/src/k8s.io/client-go/util/certificate/certificate_store_test.go#L29

that is the common way of a guardclause  and golang way :+1: